src/osx/slider_osx.cpp

```diff
@@ -11,23 +11,64 @@
 
 #if wxUSE_SLIDER
 
+#include <iostream>
```

Comment:
AFAICS this isn't really needed.

Reply:
No, I have forgotten to delete this. It has been used for debugging.